src/components/ReactTags.js

```diff
@@ -91,6 +94,13 @@ class ReactTags extends Component {
 
   constructor(props) {
     super(props);
+
+    if (!props.inline) {
```

Comment:
can you add a spec for this ?
you can stub console.error and check if the stub was called with proper arguments.

Reply:
I am getting an error if I pass inline prop. If I rename prop to anything (eg: inline123), code works fine.

Reply:
@atiqueansari1987 this error might have been thrown since you didn't stub `console.warn`. To confirm if its due to `console.warn` just comment it and try again, it shouldn't throw this error then.

Reply:
Yes stubbing console.warn no longer throws error. Thanks for the help.